.env.example

```diff
@@ -4,7 +4,7 @@
 
 # API key for the Cicero API
 # Check with the team leads for the latest Cicero API key to be used
-CICERO_API_KEY=
+CICERO_API_KEY= 
```

Comment:
This is a change that can be reverted since it doesn't have to do with the work in question. It may seem pedantic, but it is better to keep PRs as small as possible.